postgres-appliance/build_scripts/prepare.sh

```diff
@@ -30,26 +30,18 @@ fi
 # Cleanup all locales but en_US.UTF-8 and optionally specified in ADDITIONAL_LOCALES arg
 find /usr/share/i18n/charmaps/ -type f ! -name UTF-8.gz -delete
 
-# Prepare find expression for locales
-LOCALE_FIND_EXPR=(-type f)
-for loc in en_US en_GB $ADDITIONAL_LOCALES "i18n*" iso14651_t1 iso14651_t1_common "translit_*"; do
-    LOCALE_FIND_EXPR+=(! -name "$loc")
-done
-find /usr/share/i18n/locales/ "${LOCALE_FIND_EXPR[@]}" -delete
-
-# Make sure we have the en_US.UTF-8 and all additional locales available
-truncate --size 0 /usr/share/i18n/SUPPORTED
-for loc in en_US $ADDITIONAL_LOCALES; do
-    echo "$loc.UTF-8 UTF-8" >> /usr/share/i18n/SUPPORTED
-    localedef -i "$loc" -c -f UTF-8 -A /usr/share/locale/locale.alias "$loc.UTF-8"
-done
+# Dirty hack for smooth migration of existing dbs
+bash /builddeps/locales.sh
+mv /usr/lib/locale/locale-archive /usr/lib/locale/locale-archive.22
+ln -s /run/locale-archive /usr/lib/locale/locale-archive
+ln -s /usr/lib/locale/locale-archive.22 /run/locale-archive
 
 # Add PGDG repositories
 DISTRIB_CODENAME=$(sed -n 's/DISTRIB_CODENAME=//p' /etc/lsb-release)
 for t in deb deb-src; do
     echo "$t http://apt.postgresql.org/pub/repos/apt/ ${DISTRIB_CODENAME}-pgdg main" >> /etc/apt/sources.list.d/pgdg.list
 done
-curl -s -o - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+curl -s -o - https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg >/dev/null
```

Comment:
Why not just the following?
```suggestion
curl -s -o - https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg
```

Reply:
silly copy-pasting from docs...